src/modules/cmdpal/Microsoft.CmdPal.UI/ExtViews/ListPage.xaml.cs

```diff
@@ -86,49 +62,6 @@ protected override void OnNavigatedTo(NavigationEventArgs e)
         base.OnNavigatedTo(e);
     }
 
-    private async Task InitializeViewmodel(ListViewModel lvm)
-    {
-        // You know, this creates the situation where we wait for
-        // both loading page properties, AND the items, before we
-        // display anything.
-        //
-        // We almost need to do an async await on initialize, then
-        // just a fire-and-forget on FetchItems.
-        lvm.InitializeCommand.Execute(null);
-
-        await lvm.InitializeCommand.ExecutionTask!;
-
-        if (lvm.InitializeCommand.ExecutionTask.Status != TaskStatus.RanToCompletion)
-        {
-            // TODO: Handle failure case
-            System.Diagnostics.Debug.WriteLine(lvm.InitializeCommand.ExecutionTask.Exception);
-
-            // TODO GH #239 switch back when using the new MD text block
-            // _ = _queue.EnqueueAsync(() =>
-            _queue.TryEnqueue(new(() =>
-            {
-                LoadedState = ViewModelLoadedState.Error;
-            }));
-        }
-        else
-        {
-            // TODO GH #239 switch back when using the new MD text block
-            // _ = _queue.EnqueueAsync(() =>
-            _queue.TryEnqueue(new(() =>
-{
-    var result = (bool)lvm.InitializeCommand.ExecutionTask.GetResultOrDefault()!;
-
-    ViewModel = lvm;
-
-    WeakReferenceMessenger.Default.Send<NavigateToPageMessage>(new(result ? lvm : null));
-    LoadedState = result ? ViewModelLoadedState.Loaded : ViewModelLoadedState.Error;
-
-    // Immediately select the first item in the list
-    ItemsList.SelectedIndex = 0;
```

Comment:
This is the only difference in the initialization pattern here from the central one, but i think this is redundant as we have the check in the ItemsUpdated event to do this once the page is loaded...